mobile-app/app/screens/AppNavigator/screens/Balances/screens/AddOrEditAddressBookScreen.tsx

```diff
@@ -23,10 +24,12 @@ export function AddOrEditAddressBookScreen ({ route, navigation }: Props): JSX.E
   const {
     title,
     onSaveButtonPress,
-    addressLabel
+    address,
+    addressLabel,
+    isAddNew
   } = route.params
   const [labelInput, setLabelInput] = useState(addressLabel?.label)
```

Comment:
<img width="829" alt="image" src="https://user-images.githubusercontent.com/13292662/161695997-d8226060-7981-4087-a455-1ca299610d7b.png">
No issues found, but to fix the warning, it can't be `undefined` on the initial state,  `addressLabel?.label ?? ''`
If it'll break tests and take time, I think it's fine to keep the warning for now and just create a ticket

Reply:
Actually saw the warning but only on web, not on mobile. I also didnt fallback to empty string because there is a validation on clean state, and dirty state with empty string. The ideal solution is probably to use the controlled input like `react-hook-form` in Send screen